nbconvert/filters/tests/test_ansi.py

```diff
@@ -17,60 +15,57 @@ class TestAnsi(TestsBase):
     def test_strip_ansi(self):
         """strip_ansi test"""
         correct_outputs = {
-            '%s%s%s' % (TermColors.Green, TermColors.White, TermColors.Red)  : '',
-            'hello%s' % TermColors.Blue: 'hello',
-            'he%s%sllo' % (TermColors.Yellow, TermColors.Cyan) : 'hello',
-            '%shello' % TermColors.Blue : 'hello',
-            '{0}h{0}e{0}l{0}l{0}o{0}'.format(TermColors.Red) : 'hello',
-            'hel%slo' % TermColors.Green : 'hello',
-            'hello' : 'hello'}
+            '\x1b[32m\x1b[1m\x1b[0;44m\x1b[38;2;255;0;255m\x1b[;m\x1b[m': '',
+            'hello\x1b[000;34m': 'hello',
+            'he\x1b[1;33m\x1b[;36mllo': 'hello',
+            '\x1b[;34mhello': 'hello',
+            '\x1b[31mh\x1b[31me\x1b[31ml\x1b[31ml\x1b[31mo\x1b[31m': 'hello',
+            'hel\x1b[;00;;032;;;32mlo': 'hello',
+            'hello': 'hello',
+        }
 
         for inval, outval in correct_outputs.items():
             self._try_strip_ansi(inval, outval)
 
-
     def _try_strip_ansi(self, inval, outval):
         self.assertEqual(outval, strip_ansi(inval))
 
-
     def test_ansi2html(self):
         """ansi2html test"""
         correct_outputs = {
-            '%s' % (TermColors.Red)  : '<span class="ansired"></span>',
-            'hello%s' % TermColors.Blue: 'hello<span class="ansiblue"></span>',
-            'he%s%sllo' % (TermColors.Green, TermColors.Cyan) : 'he<span class="ansigreen"></span><span class="ansicyan">llo</span>',
-            '%shello' % TermColors.Yellow : '<span class="ansiyellow">hello</span>',
-            '{0}h{0}e{0}l{0}l{0}o{0}'.format(TermColors.White) : '<span class="ansigrey">h</span><span class="ansigrey">e</span><span class="ansigrey">l</span><span class="ansigrey">l</span><span class="ansigrey">o</span><span class="ansigrey"></span>',
-            'hel%slo' % TermColors.Green : 'hel<span class="ansigreen">lo</span>',
-            'hello' : 'hello'}
+            '\x1b[31m': '',
+            'hello\x1b[34m': 'hello',
+            'he\x1b[32m\x1b[36mllo': 'he<span class="ansicyan">llo</span>',
+            '\x1b[1;33mhello': '<span class="ansiyellow ansibold">hello</span>',
+            '\x1b[37mh\x1b[0;037me\x1b[;0037ml\x1b[00;37ml\x1b[;;37mo': '<span class="ansigray">h</span><span class="ansigray">e</span><span class="ansigray">l</span><span class="ansigray">l</span><span class="ansigray">o</span>',
+            'hel\x1b[0;32mlo': 'hel<span class="ansigreen">lo</span>',
+            'hello': 'hello',
```

Comment:
No, I don't think you're missing it. This wasn't possible before, so there were no tests for it.

I added a test case, but first I changed to test generators and combined the test cases for `ansi2html()` and `ansi2latex()` to avoid the annoying duplication of test strings.